SMLHelper/Initializer.cs

```diff
@@ -43,18 +55,21 @@ public static void PostPatch()
         {
             try
             {
-                Initialize();
+                PostInitialize();
             }
             catch (Exception e)
             {
                 Logger.Error($"Caught exception while trying to initialize SMLHelper{Environment.NewLine}{e}");
```

Comment:
For consistency with current recommended best practices, remove the try-catch from this method as well.  
QModManager will catch the exception, log it, and also inform the player that something went wrong.

Catching and logging like this can hide errors from the player.